types/block.go

```diff
@@ -205,19 +205,19 @@ func (h *Header) Hash() cmn.HexBytes {
 		return nil
 	}
 	return merkle.SimpleHashFromMap(map[string]merkle.Hasher{
-		"ChainID":     aminoHasher(h.ChainID),
-		"Height":      aminoHasher(h.Height),
-		"Time":        aminoHasher(h.Time),
-		"NumTxs":      aminoHasher(h.NumTxs),
-		"TotalTxs":    aminoHasher(h.TotalTxs),
-		"LastBlockID": aminoHasher(h.LastBlockID),
-		"LastCommit":  aminoHasher(h.LastCommitHash),
-		"Data":        aminoHasher(h.DataHash),
-		"Validators":  aminoHasher(h.ValidatorsHash),
-		"App":         aminoHasher(h.AppHash),
-		"Consensus":   aminoHasher(h.ConsensusHash),
-		"Results":     aminoHasher(h.LastResultsHash),
-		"Evidence":    aminoHasher(h.EvidenceHash),
+		"ChainID":         aminoHasher(h.ChainID),
+		"Height":          aminoHasher(h.Height),
+		"Time":            aminoHasher(h.Time),
+		"NumTxs":          aminoHasher(h.NumTxs),
+		"LastBlockID":     aminoHasher(h.LastBlockID),
```

Comment:
mkae sure the ordering here matches the ordering in the header

Reply:
Removed from this PR and opened an issue for it https://github.com/tendermint/tendermint/issues/1841